### ISSUE TITLE
SX OS Support

### ISSUE THREAD

**Issue by c1naeco** [exelix11/SysDVR#34]
It seems unable to get to 3006 instead of 2000, even after a steady amount of time.  
Can't find anything on the net about it, I tried to install it first to see if it changes, but it doesn't.
Tried waiting a bit, reboot CFW and to retry the whole installation process 3 times.

**Comment by exelix11** [comment]
what is your CFW ? Which version of SysDVR are you using ? Full or usb-only ?
If using the full version did you enable usb mode in the SysDVR-Config app ?

**Comment by c1naeco** [comment]
SX OS; 3.0; Full; can't go into the config app on my switch, since it won't connect. 

I just put the folders of 3.0 in the root of my sd, so that should be right

**Comment by exelix11** [comment]
As explained in the readme the releases are meant for atmosphere.
To use sysdvr on sxos you must extract the zip on your computer, rename the `atmosphere` folder to `sxos`, then open it and rename `contents` to `titles`.
After that you can copy everything to the root of your sd card and it should work.

**Comment by c1naeco** [comment]
Sadly this only makes my bootscreen freeze, hence I can't boot into the CFW. Everything still works after renaming it again. I had atmosphere as folder for some time now tho. I guess there aren't solutions for the SXOS further than this?

**Comment by exelix11** [comment]
No, afaik it should work fine, you probably have too many symodules installed and run out of ram. Try removing everything in the sxos folder (backup to pc !) on your sd and copying just SysDVR

**Comment by c1naeco** [comment]
Thanks that helped, it finally let me connect with SysDVR in the Switch and I got the 3006 ID and installed again! 

Now when I try to Stream, the next thing doesn't work. It won't connect to mpv; error number -138 (cmd command, since the client cmd is empty and doesn't load anything). 
I had a small success after trying again; It worked for half a second (and nearly gave me an heartattack by being extreamly loud, maybe deserved that for bothering you so much xd). 

Now the ClientGUI shuts down after loading so fast, that I can't capture the things that display before that.

After few tries trying to capture the problem it **FINALLY WORKED**! Thank you so much!

Sorry if I annoyed you, I am really glad you helped me this far and I hope it will help others facing the same problems! <3 If I have some money left this month I will be sure to donate it towards you <3
